### ISSUE TITLE
[Feature] Stackblitz intergration

### ISSUE THREAD

**Issue by skoropadas** [ng-doc/ng-doc#55]
### Description

I want to investigate and if possible add integration with StackBlitz for demos.

NgDoc knows the demo code, which has simplified the handling of standalone components in many ways. This will also make it easier to dynamically create StackBlitz projects where users can play around with and modify the demo code.

### Proposed solution

Enable the creation of a default configuration for the StackBlitz project's API, which will include all the necessary imports and package versions. If this configuration exists, automatically display a StackBlitz button for each demo in the guides. When the button is clicked, it will create and display the StackBlitz project.

This is just an idea for now. More information will be known once I start working on it.

### Alternatives considered

-

**Comment by pimenovoleg** [comment]
I'd really love to see this feature :) Are there any plans to implement it in the near future?

**Comment by skoropadas** [comment]
@pimenovoleg Yes, I have already started working on this task and it generally worked for stackblitz, but I thought that some users might want to work with codesandbox or some other platform, so I decided to put this task on hold for a while. But I think I will return to it and create some kind of plugins to enable integration with other platforms.

**Comment by pimenovoleg** [comment]
I think the addition of any such integration would be great, and if it's with CodeSandbox, that's fine.

**Comment by Harpush** [comment]
A button for open in stackblitz will be great
